modules/edu.gemini.seqexec.server/src/main/scala/edu/gemini/seqexec/server/GmosControllerEpics.scala

```diff
@@ -21,15 +17,16 @@ import scalaz.Scalaz._
 import scalaz.EitherT
 import scalaz.concurrent.Task
 
-object GmosControllerEpics extends GmosSouthController {
+abstract class GmosControllerEpics extends GmosController {
   private val Log = Logger.getLogger(getClass.getName)
 
+  import GmosController._
   import EpicsCodex._
 
   val CC = GmosEpics.instance.configCmd
   val DC = GmosEpics.instance.configDCCmd
 
-  override def getConfig: SeqAction[GmosSouthConfig] = ???
+  override def getConfig: SeqAction[GmosConfig] = ???
```

Comment:
A lot of work to read every value from EPICS and create a model from them. It is not used right now, but having it will allow to implement a more efficient configuration, that only changes what needs to change.